packages/x-adapter-platform/tsconfig.esm.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "module": "esnext",
+    "outDir": "./dist/esm",
+    "declaration": false,
+    "declarationDir": null,
+    "noEmit": false
```

Comment:
You can remove this `noEmit` from here. We are already emitting type declarations in the `cjs` build and we are explicitly setting `declaration: false` and `declarationDir: null`.